src/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -165,6 +165,14 @@ public static double Tanh(double value)
             return RuntimeImports.tanh(value);
         }
 
+        internal static double CopySign(double x, double y)
+        {
```

Comment:
sorry I didn't see this comment before I merge. I am thinking this can be ok to be used on Windows too and may be we consider removing the intrinsic call. if you still think we need to guard it with ifdef, I can do it in another PR